server/midddleware/middleware.go

```diff
@@ -64,7 +64,7 @@ func Get(config *config.Config) (grpc.UnaryServerInterceptor, grpc.StreamServerI
 		grpc_ratelimit.UnaryServerInterceptor(&RateLimiter{}),
 		grpc_auth.UnaryServerInterceptor(authFunction),
 		grpctrace.UnaryServerInterceptor(grpctrace.WithServiceName(config.Tags.Service)),
-		grpc_logging.UnaryServerInterceptor(grpc_zerolog.InterceptorLogger(log.Logger)),
+		//grpc_logging.UnaryServerInterceptor(grpc_zerolog.InterceptorLogger(log.Logger)),
```

Comment:
Should this be put back?

Reply:
I think probably we don't want this because this is essentially doing 2 logs per request. We need to figure out how to probably only sample request logs.

Reply:
We can do this conditionally and enable in the dev/debug environment.